compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/TextActionsTest.kt

```diff
@@ -51,7 +51,7 @@ class TextActionsTest {
             TestTextField("hello")
         }
 
-        with(rule.onNodeWithTag("tag")){
+        with(rule.onNodeWithTag("tag")) {
```

Comment:
It looks like only whitespace changed in this file

Reply:
Yes, I fixed it here and in a bunch of other test files.